src/docs/community/faq.mdx

```diff
@@ -1,36 +1,115 @@
+
+</FAQExpander>
 
-We will be continuously updating the Design Patterns section of pxblue.github.io with new patterns and code samples as they are ready. We will prioritize patterns and examples based on interest expressed from development teams. If there is something you want to see, [Contact us](/community/contactus).
+<FAQExpander question={`Do you have design patterns for _____?`}>
+
+We will be continuously updating the [Design Patterns](/patterns) section with new patterns and code samples as they are available. We will prioritize patterns and examples based on interest expressed from development teams. If there is something you want to see, please contact us.
 
 If you have built something that you would like to share, check out our information on [innersourcing](/community/innersourcing) and [Contact Us](/community/contactus) with a demo.
+
+</FAQExpander>
+
+<FAQExpander question={`Wht do other PX Blue projects look like?`}>
+
+If you are interested in seeing some examples of past and present projects that have been designed using PX Blue, please get in touch with us. We would be more than happy to point you to some reference materials from other projects.
+
+</FAQExpander>
+
+<FAQExpander question={`How much creative freedom do I have as a designer?`}>
+
+PX Blue is not meant to be proscriptive in our recommendations. We want PX Blue products to feel like they are part of the same family, but we do not want them to all feel like the same application. There are some aspects of applications that need to be the same across products to drive this consistency, but other areas are fair game for you to develop your own style. You can learn more my reading our page on [Product Identity](/design/project-identity).
+
+</FAQExpander>
+
+<FAQExpander question={`What if my development team says _____ is not possible?`}>
+
+If you are designing your project using the PX Blue assets we provide, these should be achievable without much issue on the development side. If you have specific concerns, please feel free to contact our development team and we can help your developers find a solution that will work.
+
+</FAQExpander>
+
+<FAQExpander question={`How can I contribute?`}>
+
+If you are interested in contributing to PX Blue, that's awesome! We would love to see what you have designed. If you have components or patterns that you think would be valuable to share with the rest of the PX Blue community, please get in touch with us for a discussion. You can also find more information on our [Innersourcing](/community/innersourcing#how-we-innersource-new-features) page.
+
+</FAQExpander>
+
+## Everyone
```

Comment:
Why is "for everyone" listed after "developers" and "designers"?

Reply:
🤷